src/Broadcasts/View.elm

```diff
@@ -140,7 +140,7 @@ broadcastContent time broadcast =
     let
         fractionTimeLeft : Float
         fractionTimeLeft =
-            1 + (((Date.toTime broadcast.createdAt) - time) / (24 * Time.hour))
+            1 - ((time - (Date.toTime broadcast.createdAt)) / (24 * Time.hour))
```

Comment:
also `time` is greater than `broadcast.createdAt` so I did mess up the order in the original equation